test/app.test.js

```diff
@@ -29,8 +30,8 @@ describe('Main Generator', function () {
         .withPrompts({
           version: '0.4.0',
           folder: 'temp',
-          appName: 'MEAN',
-          appDescription: 'Full-Stack JavaScript with MongoDB, Express, AngularJS, and Node.js',
+          appName: 'MEANtestAppName',
+          appDescription: 'Testing the generator',
           appKeywords: 'MongoDB, Express, AngularJS, Node.js',
           appAuthor: 'Test',
           addArticleExample: false,
```

Comment:
We should also have tests where these are true and false, and have every combination.